docker/Dockerfile.centos.7

```diff
@@ -5,4 +5,4 @@ MAINTAINER andreas-bok-sociomantic <andreas.bok@sociomantic.com>
 # install required packages
 RUN yum install -y deltarpm && \
 yum install -y rpm-build zlib-devel libtool openssl \
-    openssl-devel automake autoconf gcc-c++
+    openssl-devel automake autoconf gcc-c++ lcov
```

Comment:
Why is `lcov` used? AFAIK is just to generate coverage reports, right?

Reply:
> Why is lcov used? AFAIK is just to generate coverage reports, right?

Yes, basically just for building the coverage html report on the host. 

The tool is checked as a requirement by the autoconf macro, [m4/ax_code_coverage.m4](https://github.com/sociomantic-tsunami/libdrizzle-redux/pull/271/files#diff-9b17df31da3fcbb3b226863933edb6b2R201) since the make target `code-coverage-capture` also generates the html report.

Reply:
But if this is just to send reports to codecov, I don't think you need to "capture" the coverage yourself, codecov will take care of interpreting the coverage raw data and make the reports for you.

Reply:
True. 

ATM though getting code coverage to work for the current build configurations is a PITA. 

Will check out if I can remove the  `-X gcov` and let the codecov script do the capturing but pretty sure it will only work for compilation with `gcc`...